### ISSUE TITLE
same approach different platform? 

### ISSUE THREAD

**Issue by PhillRob** [tdwg/curriculum#3]
There has been efforts based on funding by the [JRS Biodiversity Foundation](http://jrsbiodiversity.org/), the support of the University of [Kansas Biodiversity Institute](http://biodiversity.ku.edu/) and the GBIF community tackling the gap of training materials in the field of biodiversity informatics. This has been partially published at [https://journals.ku.edu](https://journals.ku.edu).
Can I assume this aims to be a collaborative effort between TDWG, GBIF, KBI, JSR? Does this activity connect to the [GBIF interest group on biodiversity informatics training](http://community.gbif.org/pg/groups/30748/interest-group-on-biodiversity-informatics-training/)?

**Comment by HannaKoivula** [comment]
Yes, this is a collaborative effort. We have not decided on the platform yet, but the first step would be to define the terms and mechanisms for using the framework.

Fatima Parker-Allie is a member of thei TDWG group and the leader of BDIC group in GBIF.

**Comment by dschigel** [comment]
Could be useful to add a link to the TDWG pages here, I guess the charter is there at least. We also discussed that a searchable collections of resources (such as ppts) and of contacts potentially available to teach would be handy. Some kind of resource and skill buckets so when you know you need, your timebox, your budget you could use those lego block to build a perfect course.

**Comment by HannaKoivula** [comment]
Yes, this was very much wat we talked at the session in CR. We should agree how to organise GitHub and also if Github is the place we wish to eshtablish the rerource collection. I'm still learning it's functions, but maybe I can put up a draft structure and we can then discuss possible needed changes...

**Comment by debpaul** [comment]
I'd like to suggest that @iDigBio materials be added to this as well -- as part of the collaborative effort and part of the GBIF family. I'd also like to suggest we use the @theCarpentries model going forward so that all people planet-wide can contribute and access.
